### PR TITLE
"Adjust" Support for Internet Explorer [v2]

### PR DESCRIPTION
**Pull request by akaplo**
Ionic 2 technically doesn't support IE, so this makes it a little clearer in the following ways.

1. Using the ES7 `Array.prototype.includes()` was causing crashes, so I switched to lodash's `_.includes()`.
2. When the app loads (regardless of which page), you get the following toast for 10 seconds
image

3. `<ion-select>` doesn't work at all in IE.  We run into this issue on `Stop`, `Routes and Stops`, `Plan Trip`, and `Settings`.

- On `R&S` and `Stop`, we simply remove the bar with the `<select>`

image

image

- On `Plan Trip` and `Settings`, we tell the user that this feature isn't supported.

image

image

### CODE REVIEW

File: src/pages/stop/stop.html [hunk 1]
```diff
@@ -24,95 +24,88 @@
...
-           </ion-grid>
-       </div>
-     </div>
-   </div>
-     <div *ngIf="order == '1'">
-       <div ion-item *ngFor="let direction of departuresByTime"
-        (click)="goToRoutePage(direction.RouteId)"
-        [style.background-color]="'#'+routeList[direction.RouteId]?.Color"
-        [style.color]="'white'">
+      <div *ngFor="let direction of departuresByDirection">
```
Comment (akaplo) [inline src/pages/stop/stop.html:30]: Everything below this is just indentation fixes.

File: src/pages/plan-trip/plan-trip.html [hunk 2]
```diff
@@ -30,7 +30,10 @@
         type="text" placeholder="Enter a destination" [(ngModel)]="request?.destination.name"/>
       </label>
     </ion-item>
-    <ion-item>
+    <ion-item *ngIf="isInternetExplorer">
+      <p style="font-style: italic;">Internet Explorer only supports finding buses leaving now.</p>
```
Comment (werebus) [inline src/pages/plan-trip/plan-trip.html]: Slight grammatical ambiguity: the buses are leaving now, but this sentence could also be read to imply that IE now only supports finding leaving buses.

"buses **that are** leaving now", perhaps?

**Review by werebus — APPROVED** [review]
This seems like a perfectly resonable attempt at IE support. It's OK for IE to be a slightly worse experience, but it should be mostly usable.

File: src/pages/plan-trip/plan-trip.html (hunk 2 above)
Comment (akaplo) [reply src/pages/plan-trip/plan-trip.html]: Fair point. While we're on the subject, I realized that "leaving now" makes little sense without the context that there **was** another option.

I propose `Internet Explorer does not support searching the schedule on specific dates and times, and will only find the soonest available trip.`.

**Pull request merged**